Assets/Scripts/Combat/Abilities/ItemSlot.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+// This handles the GameObject that holds DragAndDrop items
+// When a GameObject is dropped on it, it will snap to the center
+public class ItemSlot : MonoBehaviour, IDropHandler
+{
+    private int sweetsDropped = 0;
+    private int rotsDropped = 0;
+    private bool isInside;
```

Comment:
This is not being used either. Can be removed.

Reply:
It gets used in another script (Confection.cs), that's how I keep track of what was dropped in the pot